models/info.js

```diff
@@ -0,0 +1,42 @@
+module.exports = (sequelize, DataTypes) => {
+  return sequelize.define('info', {
+    id: {
+      type: DataTypes.INTEGER,
+      autoIncrement: true,
+      primaryKey: true
+    },
+    product_id: {
+      type: DataTypes.INTEGER
+    },
+    rating: {
+      type: DataTypes.INTEGER
+    },
+    date: {
+      type: DataTypes.BIGINT
+    },
+    summary: {
+      type: DataTypes.TEXT
+    },
+    body: {
+      type: DataTypes.TEXT
+    },
+    recommended: {
+      type: DataTypes.BOOLEAN
+    },
+    reported: {
+      type: DataTypes.BOOLEAN
+    },
+    reviewer_name: {
+      type: DataTypes.TEXT
+    },
+    reviewer_email: {
```

Comment:
Would it be better to save space for the name and email fields and use the Varchar/String data types instead? Since we probably are not expecting a large body of text.